wmfdata/spark.py

```diff
@@ -65,15 +65,8 @@
 
 def get_active_session():
     """
-    DEPRECATED: Returns the active session if there is one and None otherwise.
```

Comment:
In my experience, you can do a lot of analysis with Pyspark just using the methods attached to the `SparkSession` object, without ever importing `pyspark` directly (for example, I never imported it directly in [this big notebook](https://gitlab.wikimedia.org/nshahquinn-wmf/editor-month-history-shift/-/blob/main/analysis.ipynb)).

However, if you start the session by using `wmf.spark.run`, you aren't returned that session object. In that case, if you want to start use the PySpark APIs with the existing session, you would have to do: `import pyspark; spark = pyspark.sql.SparkSession.getActiveSession()`. Obviously, not a big deal to do, but I do always find myself having to look up the exact command, whereas I find `spark = wmf.spark.get_active_session()` very ergonomic and obvious because it's parallel to `spark = wmf.spark.create_session()`. 

I'm generally against wrappers for simple operations; that's why I deprecated it in the first place. But after I did that, there were several instances where I found myself using and appreciating it, so now I feel like this particular wrapper is useful.

Reply:
Makes sense.